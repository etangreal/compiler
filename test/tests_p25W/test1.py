x = True
y = True

while(x and y):
	print 100
	x = False


	if ( True ):
		print 33

		if ( True ):
			print 22
			x  = True
			while(x and y):
				print 1
				x = 0
				y = 0

		elif(True):
			print 1000

		else:
			print 1