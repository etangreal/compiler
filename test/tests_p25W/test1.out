100
33
22
1
